playground/models.py

```diff
@@ -1,16 +1,31 @@
-from sqlalchemy.ext.declarative import declarative_base
+import sqlalchemy  as sa
+from sqlalchemy.ext.declarative import declarative_base, declared_attr
 from sqlalchemy.orm import (
     scoped_session,
     sessionmaker,
-)
+    relationship)
+from sqlalchemy_utils import UUIDType
 from zope.sqlalchemy import register
 
 from keyloop.ext.sqla.identity import Identity
+from keyloop.ext.sqla.permission import Permission
 
 DBSession = scoped_session(sessionmaker())
 register(DBSession)
 Base = declarative_base()
 
+permission_grant_table = sa.Table(
+    'permission_grant', Base.metadata,
+    sa.Column('identity_id', UUIDType, sa.ForeignKey('identity.id')),
+    sa.Column('permission_id', sa.Integer, sa.ForeignKey('permission.id'))
+)
+
```

Comment:
espera - eu não tava entendo bem o conceito de "permission grant" e só entendi aqui - é a "instância de uma permissão para um identity" . Legal - fui ver por que não tinha entendido - é por que no Schema de PermissionGrant não aparece  Identity.

( A interface está certa. Mas temos que pensar que no momento, a interface tem que refletir o Schema mais por conta de não termos uma ferramenta que gere a interface sozinha a partir do Schema )


Temos que ter em mente que  e o Schema, no contexto do Keyloop, é o nosso "model" . Lá é que tem que aparecer todos os campos e tudo que for relacionado a uma identidade - se para um deerminado endpoint não precisamos de alguns dos campos, ou precisamos de campos a mais, tem que usar as lista `include`  e `exclude`

Então se todos os endpoints de permission tem uma identity implícita, e acessam o permission por nome, não por ID, isso tem que se rrefletido nso endpoints  - mas o Schema tem que ser algo que eu bata o olho e entenda que esse objeto faz o relacionamento de uma Permission pra um Idenitity - essa informação não tem que ficar nas implementações de referência (ext.sqla)  e menos ainda no playground.